content/pom.xml

```diff
@@ -241,6 +241,52 @@
                             </execution>
                         </executions>
                     </plugin>
+                    <plugin>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <phase>prepare-package</phase>
+                                <configuration>
+                                    <tasks>
+                                        <replace file="target/jcr_root/libs/core/wcm/components/breadcrumb/v2/breadcrumb/.content.xml" token=".core-wcm" value="Core Structure"/>
```

Comment:
A bit late to the party but wouldn't it make sense to define Maven properties for the 3 group names? Also using Maven filtering instead of falling back to using Ant feels more natural in Maven, that would also allow to not needing to extend this list every time a new component is added. That way the default property value could be the On Prem/AMS component name, while for cloud you override the property values in a dedicated Maven Profile.
@vladbailescu WDYT?

Reply:
I agree. The way it is implemented right now, feels like a nasty hack. Maven filtering is so much nicer.

Reply:
@kwin @vladbailescu should I create a new ticket to track this?

Reply:
@wimsymons, @kwin , we already tried that approach in #1216 but we weren't able to get it work the way we want: we would like to be able to generate both regular (with hidden component groups) and `cloud` (with visible component groups) artifacts in the same build, if the `cloud` profile is active (it helps a lot in the release process). If you think of a better way to do it than the way I currently set it up, we'd be grateful to get a PR for your idea and test it out. You can reference #1274 in the PR.

Reply:
@vladbailescu Actually that should be possible with the parameter `filterFiles` which allows you to reference two different java properties files for cloud and classic. That way replacement should work the way you intend.